src/main/java/duke/query/EventQueryHandler.java

```diff
@@ -0,0 +1,62 @@
+package duke.query;
+
+import duke.DukeException;
+import duke.task.Task;
+import duke.task.TaskTracker;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The EventQueryHandler class handles user queries for adding new events.
+ */
+public class EventQueryHandler extends DeadlineQueryHandler {
+    public EventQueryHandler(TaskTracker tt) {
+        super(tt);
+    }
+
+    /**
+     * Processes a query for adding an event.
+     *
+     * @param query a user input string
+     * @return response from adding an event
+     * @throws DukeException
+     */
+    @Override
+    public String processQuery(String query) throws DukeException {
+        String[] parsed = QueryParser.parseQuery(query, new String[]{"/from", "/to"});
+        String desc = parsed[1];
+        if (desc == null || desc.isBlank()) {
```

Comment:
I like the naming of the `isBlank()` function, to convey the intention of the function!